packages/web-frontend/src/containers/EnlargedDialog/EnlargedDialog.js

```diff
@@ -266,9 +264,7 @@ const EnlargedDialogComponent = ({
 
   const { doExport } = useChartDataExport(viewContentWithExportOptions, exportTitle);
 
-  const onExport = async format => {
-    setExportStatus(STATUS.ANIMATING);
-    await sleep(100); // allow some time for the chart transition to finish before hiding the loader
+  const onExport = async (format, exportRef) => {
```

Comment:
exportRef is now passed in to EnlargedDialog which passes it on to the png util. All export stuff now lives in ExportDialog.